src/renderer/remote-loader.ts

```diff
@@ -115,13 +126,40 @@ export class RemoteLoader {
     }
   }
 
+  public async getGistRevisions(gistId: string): Promise<GistRevision[]> {
+    console.log('getGistRevisions!');
+    try {
+      const octo = await getOctokit(this.appState);
+      const revisions = await octo.gists.listCommits({
+        gist_id: gistId,
+      });
+
+      console.log('Gist revisions', revisions.data);
+
+      return revisions.data.map((r) => ({
+        sha: r.version,
+        date: r.committed_at,
+        changes: r.change_status,
+      }));
```

Comment:
suggestion: Should we reverse this list? Intuitively, I would have guessed that "Revision 3" was the latest one and "Revision 1" was the oldest, but it seems like this is reversed in the actual.

<img width="497" height="416" alt="image" src="https://github.com/user-attachments/assets/f6ae4398-45d3-40ed-99fa-450bee82c85f" />